docs/tree-entities.md

```diff
@@ -187,9 +188,16 @@ const a112 = new Category()
 a112.name = "a112"
 a112.parent = a11
 await dataSource.manager.save(a112)
+
+// UPDATE
+const updateA1 = await dataSource.manager.findOneBy({ id })
+updateA1.name = "updateA1"
+updateA1.parent = a1
+updateA1.children = [a11, a12]
+await dataSource.manager.save(updateA1)
 ```
 
-To load such a tree use `TreeRepository`:
+To load such a tree use `TreeRepository` example `TreeRepository<Entity>`:
```

Comment:
what does it mean example `TreeRepository<Entity>` ?

Reply:
oh, I try to example use TreeRepository on typescript. I'm  trying to find  to use  ```TreeRepository``` I think maybe I put example to using